bbp/nse/bluepy/default.nix

```diff
@@ -18,7 +19,7 @@ let
     info-version-0_9_6 = {
         version = "0.9.6";
         rev = "cf80c78ab2a5d6b0c87fcd75c7692b088d0587c6";
-        sha256 = "0ldn5hmgfyyslpvwyvch27y785jn73mjw15dj7idh1s8dkvacc29";
+        sha256 = "0i80fs37cs8930b1vb86h89f9gp6hpq0xx7cylyak58qq29wyqs8";
```

Comment:
It seems 0.9.6 derivation build was not working, at least with the recent `bbp-nixpkgs` master.
Maybe it's a sign that I can just remove it?

Reply:
I would very surprise it does not build, because jenkins build it on each modification. The target to trigger if you want to build it is -A modules.bluepy

I would prefer we do not remove it completely, in case some users still have scripts that depend of that with the old API.

However, you can swap it as "bluepy-> bluepy_legacy; bluepy_latest -> bluepy", and if we do nto see any usage or complain in the incoming months through the module logger system, we can definitively rip it off.

Reply:
> The target to trigger if you want to build it is -A modules.bluepy
This one triggers bluepy==0.6.1
The one associated with bluepy==0.9.6 was `modules.bluepy_dev`, maybe it's not included in  the list of the ones triggered by Jenkins?
Anyway, I'll change module name for both 0.6.1 and 0.9.6 to 'bluepy-legacy' as you've suggested (I can use the same name for both, right?).

Reply:
> I can use the same name for both, right?

I mean module name, not the derivation.
I'll add a commit to make the intention more clear.

Reply:
It's fine, their version is different, it will create two modules. 

I have added them in the nse_legacy module set to build them by default